tests/cypress/integration/api/admin/user.spec.ts

```diff
@@ -1,27 +1,32 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import gql from 'graphql-tag'
 import { apollo } from '../../../support/apollo'
+import { DocumentNode } from 'graphql'
 
 describe('Test admin user endpoint', () => {
+    let GQL_USER: DocumentNode
+
+    before('load graphql file', function () {
+        GQL_USER = require(`graphql-tag/loader!../../../fixtures/admin/user.graphql`)
+    })
+
     it('gets a user', () => {
         cy.apolloQuery(
             apollo(Cypress.config().baseUrl, { username: 'root', password: Cypress.env('SUPER_USER_PASSWORD') }),
             {
-                query: gql`
-                    {
-                        admin {
-                            userAdmin {
-                                user(userName: "root") {
-                                    name
-                                }
-                            }
-                        }
-                    }
-                `,
+                query: GQL_USER,
```

Comment:
maybe you could use the GQL_USER query in other tests  ?

Reply:
will do it now